nipype/pipeline/engine/utils.py

```diff
@@ -1294,6 +1296,51 @@ def write_workflow_prov(graph, filename=None, format='all'):
     return ps.g
 
 
+def write_workflow_resources(graph, filename=None):
+    import simplejson as json
+    if not filename:
+        filename = os.path.join(os.getcwd(), 'resource_monitor.json')
+
+    big_dict = {
+        'time': [],
```

Comment:
is `time` different from `duration`?

Reply:
yes, time is the result of `time.time()`. Duration is `time.time()` at start - `time.time()` at end.